From c181367bcfac0570f5965d21c4a40b49939a53c9 Mon Sep 17 00:00:00 2001
From: Christopher Albert <albert@tugraz.at>
Date: Wed, 22 Apr 2026 12:28:01 +0200
Subject: [PATCH 05/23] testsuite: Add Bugzilla reporter attribution to pr92613
 tests [PR fortran/92613]

Retrospective attribution header added as part of a manual
Bugzilla-provenance audit on 2026-04-22 (meta-repo tracking commit
40f0a94; GitHub issues on krystophny/gcc-dev).  The existing GCC
Fortran testsuite convention (see pr113363.f90, pdt_50.f03,
arrayio_4.f90) uses an in-file "! Contributed by ..." line; practice
is inconsistent and attribution is not strictly required for small
Bugzilla reducers, but we are adding it retroactively for provenance
safety and completeness.  No behavioural change.

Assisted-by: Claude (Anthropic)
Assisted-by: GPT (OpenAI)

	PR fortran/92613

gcc/testsuite/ChangeLog:

	* gfortran.dg/pr92613.f90: Add reporter attribution header.
	* gfortran.dg/pr92613_2.f90: Add reporter attribution header.

Signed-off-by: Christopher Albert <albert@tugraz.at>
---
 gcc/testsuite/gfortran.dg/pr92613.f90   | 1 +
 gcc/testsuite/gfortran.dg/pr92613_2.f90 | 1 +
 2 files changed, 2 insertions(+)

diff --git a/gcc/testsuite/gfortran.dg/pr92613.f90 b/gcc/testsuite/gfortran.dg/pr92613.f90
index 78b462385d1..f2400e4ebc3 100644
--- a/gcc/testsuite/gfortran.dg/pr92613.f90
+++ b/gcc/testsuite/gfortran.dg/pr92613.f90
@@ -3,6 +3,7 @@
 # 1 "<command-line>"
 # 1 "<test>"
 ! PR fortran/92613
+! Contributed by Ignacio Fdez. Galvan  <jellby@yahoo.com>
 ! { dg-do compile }
 ! { dg-options "-cpp -fpreprocessed" }
 program test
diff --git a/gcc/testsuite/gfortran.dg/pr92613_2.f90 b/gcc/testsuite/gfortran.dg/pr92613_2.f90
index 8706a1dc8e6..d2e693b9e82 100644
--- a/gcc/testsuite/gfortran.dg/pr92613_2.f90
+++ b/gcc/testsuite/gfortran.dg/pr92613_2.f90
@@ -3,6 +3,7 @@
 # 1 "<command-line>"
 # 1 "<test>"
 ! PR fortran/92613
+! Contributed by Ignacio Fdez. Galvan  <jellby@yahoo.com>
 ! { dg-do preprocess }
 ! { dg-options "-cpp -fpreprocessed" }
 ! { dg-error ".-E. is not supported with .-fpreprocessed." "" { target *-*-* } 0 }
-- 
2.53.0

